src/services/Hooks/useCheckPermissions.ts

```diff
@@ -0,0 +1,41 @@
+import * as Notifications from 'expo-notifications';
+import * as Permissions from 'expo-permissions';
```

Comment:
this isn't a big deal but this file has one function that looks to prepare constants depending on scenario, and then another api function. Like mentioned earlier, this isn't really a hook, but just a normal api function call, so you could combine these and move this file into an api folder

Reply:
ok